MetricsDashboard.WebApi/Controllers/TilesController.cs

```diff
@@ -13,15 +13,30 @@ public class TilesController : ControllerBase
     {
         private readonly ITileService _tileService;
 
-        public TilesController(ITileService tileService)
+        private readonly ITileDataService _tileDataService;
+
+        public TilesController(ITileService tileService, ITileDataService tileDataService)
         {
             _tileService = tileService;
+            _tileDataService = tileDataService;
         }
 
         [HttpGet("")]
         public async Task<IList<TileDto>> GetAll(CancellationToken cancellationToken)
         {
             return await _tileService.GetAllTilesAsync(cancellationToken);
         }
+
+        [HttpPost("metric")]
+        public async Task SaveMetric(SaveValueDto<decimal> saveValueDto, CancellationToken cancellationToken)
```

Comment:
You've moved those methods from TilesDataController, so maybe you should join also TileService and TileDataService?
I've done it in separate controllers because metric and tile is something else and wanted to service to be REST